src/orders/orders.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Types } from 'mongoose';
+import { CreateOrderDto } from './dto/order.create.dto';
+import { OrdersService } from './orders.service';
+import {
+  Body,
+  Controller,
+  Get,
+  Param,
+  Patch,
+  Post,
+  HttpException,
+  Query,
+} from '@nestjs/common';
+
+@Controller('orders')
+export class OrdersController {
+  constructor(private ordersService: OrdersService) {}
+
+  @Get() // Request type : GET /api/v1/orders
+  get_orders() {
+    return this.ordersService.get_orders();
+  }
+
+  @Post() // Request type : POST /api/v1/orders
+  create_order(@Body() createOrderDto: CreateOrderDto) {
+    return this.ordersService.create_order(createOrderDto);
+  }
+
+  @Patch('/:id') // Request type : PATCH /api/v1/orders/:id with status param (example : /api/v1/orders/ObjectID?status=Cooking)
+  update_status_order(
+    @Param('id') id: string,
+    @Query('status') status: string,
+  ) {
```

Comment:
Should be using JSON content type instead of query parameters for the PATCH method.
The payload would be more expressive and more readable.
See RFC 6902.

Reply:
update into this now. Please see in lastest commit in feature/orders branch
```ts
@Patch() // Request type : PATCH /api/v1/orders
update_status_order(@Body() new_status: UpdateOrderDto) {
  return this.ordersService.update_order_status(new_status);
}
```